webpack.config.js

```diff
@@ -15,10 +14,18 @@ module.exports = {
             loader: 'babel-loader',
         }],
     },
+    plugins: [
+        new webpack.DefinePlugin({
+            'process.env.NODE_ENV': JSON.stringify(process.env.NODE_ENV),
+        }),
+    ].concat(production ?  [
+        new webpack.optimize.UglifyJsPlugin(),
```

Comment:
@jeromemacias I rather like command line argument in this precise case. It makes the config file heavier to read for non meaningful information: it is well known that production file are minified.

But true, if we need some specific options, we'll need it.